### PR TITLE
UCT/IB/TEST: Support non-inline KLM list

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7953/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7955/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7959/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@Artemy-Mellanox @hoopoepg pls take a look

File: test/gtest/uct/ib/test_ib_md.cc [hunk 1]
```diff
@@ -31,19 +32,32 @@ void test_ib_md::ib_md_umr_check(void *rkey_buffer,
...
+    if (ucs_get_phys_mem_size() < size * 8) {
+        UCS_TEST_SKIP_R("not enough physical memory");
+    }
+    if (ucs_get_memfree_size() < size * 4) {
+        UCS_TEST_SKIP_R("not enough free memory");
+    }
+
+    buffer     = NULL;
+    alloc_size = size;
+    status = ucs_mmap_alloc(&alloc_size, &buffer, 0
```
Comment (hoopoepg) [inline test/gtest/uct/ib/test_ib_md.cc]: minor: align '='

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7963/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7964/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7973/ for details (Mellanox internal link).

**Pull request merged**